src/utils/verordening.ts

```diff
@@ -0,0 +1,372 @@
+
+    return typeof geoValue === 'string'
+        ? geoValue
+        : typeof geoValue === 'object' &&
+          !Array.isArray(geoValue) &&
+          geoValue !== null &&
+          geoValue?.UUID !== undefined
+        ? geoValue.UUID
+        : null
+}
+
+/**
+ * @returns A promise that resolves to a VerordeningenRead with type Lid
+ */
+export const createVerordeningLid = async (
+    lid: Partial<
+        Omit<VerordeningenWrite, 'Gebied'> & { Gebied?: string | null }
+    >
+) => {
+    return await postVerordeningSection({
+        Type: 'Lid' as const,
+        Status: 'Vigerend' as const,
+        Volgnummer: '',
+        ...lid,
+    })
+}
+
+export const replaceReorderedSections = (
+    indexPath: number[],
+    newLineageClone: VerordeningLineageRead,
+    reorderedSections: VerordeningStructureChild[]
+) => {
+    if (
+        indexPath.length === 0 &&
+        newLineageClone?.Structuur?.Children !== undefined
+    ) {
+        newLineageClone.Structuur.Children = reorderedSections
+    } else if (
+        indexPath.length === 1 &&
+        newLineageClone?.Structuur?.Children[indexPath[0]]?.Children !==
+            undefined
+    ) {
+        newLineageClone.Structuur.Children[indexPath[0]].Children =
+            reorderedSections
+    } else if (
+        indexPath.length === 2 &&
+        newLineageClone?.Structuur?.Children[indexPath[0]]?.Children[
+            indexPath[1]
+        ]?.Children !== undefined
+    ) {
+        newLineageClone.Structuur.Children[indexPath[0]].Children[
+            indexPath[1]
+        ].Children = reorderedSections
+    } else if (
+        indexPath.length === 3 &&
+        newLineageClone?.Structuur?.Children[indexPath[0]]?.Children[
+            indexPath[1]
+        ]?.Children[indexPath[2]]?.Children !== undefined
+    ) {
+        newLineageClone.Structuur.Children[indexPath[0]].Children[
+            indexPath[1]
+        ].Children[indexPath[2]].Children = reorderedSections
+    }
+
+    return newLineageClone
+}
+
+export const mutateVerordeningenReadToVerordeningenWrite = (
+    readObject: VerordeningenRead
+) => {
+    const writeObject: VerordeningenWrite = {}
+
+    const verordeningenWriteProperties = [
+        'Begin_Geldigheid',
+        'Eind_Geldigheid',
+        'Inhoud',
+        'Status',
+        'Titel',
+        'Type',
+        'Volgnummer',
+        'Weblink',
+    ] as const
+
+    const inlinedProperties = [
+        'Eigenaar_1',
+        'Eigenaar_2',
+        'Portefeuillehouder_1',
+        'Portefeuillehouder_2',
+        'Opdrachtgever',
+        'Gebied',
+    ] as const
+
+    verordeningenWriteProperties.forEach(property => {
+        writeObject[property] = readObject[property]
+    })
+
+    inlinedProperties.forEach(property => {
+        writeObject[property] = readObject[property]?.UUID
+    })
+
+    return writeObject
+}
+
+export const patchVerordeningSection = (
+    values: VerordeningenWrite,
+    lineageID: number
+) =>
+    axios
```

Comment:
@Aidenbuis I believe these queries exist in the swagger specs now. So they should be available in the fetchers file when you generate new types.

Reply:
These don't exist on the swagger spec due to the API code of the verordening being more custom then the other objects. I asked Jordy if they can implement these in the future.